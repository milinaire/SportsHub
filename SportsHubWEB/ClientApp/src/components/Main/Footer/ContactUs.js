import React, {Component, Fragment} from "react";
import "./Footer.css";
import {BaseLayout} from "../Layout/BaseLayout";

export class ContactUs extends Component {
  state = {
    Headline: "some headline from db",
    Address: "address from db",
    Tel: "Tel from db",
    Email: "Email from db",
  };
  render() {
    return (

      <Fragment>
        <BaseLayout>
          <div>
            <h1>{this.state.Headline}</h1>
            <p>{this.state.Address}</p>
            <p>{this.state.Tel}</p>
            <p>{this.state.Email}</p>
          </div>
        </BaseLayout>
      </Fragment>
    );
  }
}
