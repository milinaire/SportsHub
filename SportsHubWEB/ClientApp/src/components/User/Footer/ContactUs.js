import React, {Component, Fragment} from "react";
import "./Footer.css";

export class ContactUs extends Component {
  state = {
    Headline: "some headline from db",
    Address: "address from db",
    Tel: "Tel from db",
    Email: "Email from db",
  };

  render() {
    return (
      <Fragment>
        <div className="main-part-footer">
          <h1>{this.state.Headline}</h1>
          <p>{this.state.Address}</p>
          <p>{this.state.Tel}</p>
          <p>{this.state.Email}</p>
        </div>
      </Fragment>
    );
  }
}
