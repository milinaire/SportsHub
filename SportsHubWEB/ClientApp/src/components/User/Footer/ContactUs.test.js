
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import {ContactUs} from "./ContactUs";



describe('Main\Footer\ContactUs test', () => {
  it('Main\Footer\ContactUs should match snapshot', () => {
    const component = renderer.create(<ContactUs/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
