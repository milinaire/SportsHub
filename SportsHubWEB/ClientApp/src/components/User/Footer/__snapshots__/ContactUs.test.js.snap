// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooterContactUs test MainFooterContactUs should match snapshot 1`] = `
<div
  className="main-part-footer"
>
  <h1>
    some headline from db
  </h1>
  <p>
    address from db
  </p>
  <p>
    Tel from db
  </p>
  <p>
    Email from db
  </p>
</div>
`;
